android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/models/MeasureReportPopulationResult.kt

```diff
@@ -19,12 +19,14 @@ package org.smartregister.fhircore.quest.ui.report.measure.models
 import androidx.compose.runtime.Stable
 import kotlinx.serialization.Serializable
 
+const val MEASURE_REPORT_DENOMINATOR_MISSING = -999999999
+
 @Stable
 @Serializable
 data class MeasureReportPopulationResult(
   val title: String = "",
   val count: String = "",
   val dataList: List<MeasureReportIndividualResult> = emptyList(),
   val indicatorTitle: String = "",
-  val measureReportDenominator: Int = 0
+  val measureReportDenominator: Int = MEASURE_REPORT_DENOMINATOR_MISSING
```

Comment:
can we set this to null and have that be the check? I'd worry that someone does math w/this not realizing it's a sentinel